src/mlj_interface.jl

```diff
@@ -94,15 +94,20 @@ function MMI.fit(m::KMeans, verbosity::Int, X)
 
     # fit model and get results
     verbose = verbosity > 0  # Display fitting operations if verbosity > 0
-    fitresult = ParallelKMeans.kmeans(algo, DMatrix, m.k;
+    result = ParallelKMeans.kmeans(algo, DMatrix, m.k;
                                       n_threads = m.threads, k_init=m.k_init,
                                       max_iters=m.max_iters, tol=m.tol, init=m.init,
                                       verbose=verbose)
 
+    cluster_labels = MMI.categorical(1:m.k)
+    fitresult = (result.centers, cluster_labels, result.converged)
```

Comment:
I think we can go with `NamedTuples`
```
fitresult = (centers = result.centers, labels = cluster_labels, converged = result.converged)
```

It should not affect those who want to use indices, but will be much more convenient to manipulate.